README.md

```diff
@@ -14,29 +13,26 @@
 - [Discussion Forums](#discussion-forums)
 - [Contribute](#contribute)
 
-
-
 ## Applications
 
-
 ### Chat Clients
 
-
 - [ChitChat](https://github.com/stonesam92/ChitChat) - A native Mac app wrapper for WhatsApp Web. ![Open-Source Software][OSS Icon]
 - [LimeChat](http://limechat.net/mac/) - Instant messaging application. ![Open-Source Software][OSS Icon]
 - [Messenger for Desktop](http://messengerfordesktop.com/) - An app for Facebook messenger. ![Open-Source Software][OSS Icon]
 - [Textual 5](https://www.codeux.com/textual/) - An Internet Relay Chat (IRC) client. ![Open-Source Software][OSS Icon]
 
-
```

Comment:
The basic trend is to leave 2 newlines before starting a new category or sub category to make it more readable, you should stick to that.